scripts/qa_crowbarsetup.sh

```diff
@@ -2068,6 +2068,72 @@ function onadmin_get_ip_from_dhcp
         END{ if (res=="") exit 1; print res }' $leasefile
 }
 
+function onadmin_lonely_node_sshkey
```

Comment:
The prefix `onadmin_` is meant for functions that should be called from **other** places/hosts than the admin host. So from within mkcloud it is called like `onadmin new_function`.

Functions that are only called from within qa_crowbarsetup and that are not meant to be called from other places should not carry this prefix.